packages/gestalt/src/Text.js

```diff
@@ -122,7 +122,15 @@ export default function Text({
 }: Props): Node {
   let colorClass = null;
   const colorName = semanticColors.includes(color) ? `${color}Text` : color;
-  if (allowedColors.includes(color)) {
+  if (
+    allowedColors.includes(color) &&
+    colorName !== 'dark' &&
```

Comment:
Note that `!['dark', 'error', 'light', 'subtle', 'success', 'warning'].includes(colorName)` doesn't work